src/Sdk/DTPipelines/Pipelines/ObjectTemplating/PipelineTemplateEvaluator.cs

```diff
@@ -40,7 +40,7 @@ public class PipelineTemplateEvaluator
         /// <summary>
         /// Gets the maximum error message length before the message will be truncated.
         /// </summary>
-        public Int32 MaxErrorMessageLength => 500;
+        public Int32 MaxErrorMessageLength { get; set; } = 500;
```

Comment:
any reason to not default this to `int.max`?

Reply:
safe defaults for the server

Reply:
(whereas the fixes i'm adding are on the client)